cmd/import.go

```diff
@@ -57,8 +57,6 @@ var (
 		"collection exists. use --append if you need to add documents to existing collection")
 
 	ErrNoRecordsExpected = fmt.Errorf("no records expected in the collection after fixing numbers")
-
```

Comment:
Not needed as the fixing will anyway not happen if the collection has existing data.